fix: Update the jobs readme description (#796)
medusajs/medusa-starter-default: src/jobs/README.md

> - A configuration object defining the job. It has two properties:

Reviewer: ```suggestion
- A configuration object defining the job. It has three properties:
```

  - `name`: a unique name for the job.
  - `schedule`: a [cron expression](https://crontab.guru/).
  - `numberOfExecutions`: an optional integer, specifying how many times the job will execute before being removed

> The `handler` is a function which takes one parameter, `container`, a `MedusaContainer` instance which can be used to resolve services.

Reviewer: ```suggestion
The `handler` is a function that accepts one parameter, `container`, which is a `MedusaContainer` instance used to resolve services.
```
